src/lib/jsonParseV2/parseAttributeValue.cpp

```diff
@@ -49,7 +49,8 @@ std::string parseAttributeValue(ConnectionInfo* ciP, ContextAttribute* caP)
   if (document.HasParseError())
   {
     alarmMgr.badInput(clientIp, "JSON parse error");
-    oe.fill(SccBadRequest, "Errors found in incoming JSON buffer");
+    oe.reasonPhrase = ERROR_STRING_PARSERROR;
```

Comment:
This could be done with a new fill() method, but not sure if just one case (this one) justifies the creation of the new method...